src/scripts/pokeDetail.js

```diff
@@ -0,0 +1,45 @@
+document.addEventListener('DOMContentLoaded', async function () {
+  const params = new URLSearchParams(window.location.search);
+  const pokemonName = params.get('name');
+
+  const pokemonImage = document.getElementById('pokemonImage');
+  const pokemonNameElement = document.getElementById('pokemonName');
+  const pokemonAbilities = document.getElementById('pokemonAbilities');
+  const prevButton = document.getElementById('prevButton');
+  const nextButton = document.getElementById('nextButton');
+  const goToListButton = document.getElementById('goToListButton');
+
+  async function showPokemonData(pokemonName) {
+    try {
+      const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`);
+      const data = await response.json();
+
+      pokemonImage.src = data.sprites.front_default;
+      pokemonNameElement.textContent = `Name: ${data.name}`;
+
+      const abilities = data.abilities.map(ability => ability.ability.name);
+      pokemonAbilities.textContent = `Abilities: ${abilities.join(', ')}`;
+
+      // Fetch the next Pokemon details
+      const nextResponse = await fetch(`https://pokeapi.co/api/v2/pokemon/${data.id + 1}`);
```

Comment:
@Kamildeeal please create helper functions for the api that will handle adding BASE_API_URL and unwraping the data so you could call it like this

```
  const response = await Api.get(`/v2/pokemon/${data.id + 1}`);
```

Reply:
because right now you need to reach time repeat ` await fetch` and `awat response.json()`. We need abstract functions that will handle that